store/postgres_migration.go

```diff
@@ -204,11 +187,16 @@ func createFieldStatement(tenant string, info tableInfo, fieldInterface interfac
 func createJoinStatement(tenant string, info tableInfo, joinInterface interface{}) (string, error) {
 	join, ok := (joinInterface).(core.TableJoin)
 	if !ok {
-		return "", fmt.Errorf("uniqueInterface not assignable to bool: %s", joinInterface)
+		return "", fmt.Errorf("cannot assign type %s to core.TableJoin", reflect.TypeOf(joinInterface).String())
```

Comment:
❓ The canonical logging format would be `"cannot assign type to core.TableJoin: %s"`, no?